### PR TITLE
Add more article strips

### PR DESCRIPTION
**Pull request by VWSCoronaDashboard1**
## Summary

- Add article strips to nursing home, infectious people, elderly at home and disability care pages.
- Add extra article strip to GGD section of positive tests page.

### Governance

- [ ] Documentation is added
- [ ] Test cases are added or updated
- [ ] I've read the contributing document https://github.com/minvws/.github/blob/master/CONTRIBUTING.md
- [ ] I've read and understand the Code of Conduct https://github.com/minvws/.github/blob/master/CODE_OF_CONDUCT.md
- [ ] I understand that any contributions or suggestions I made may make it into the actual code. I've read the License
      https://github.com/minvws/nl-covid19-notification-app-coordination/blob/master/LICENSE.txt
      and the contributor license agreement https://github.com/minvws/nl-covid19-notification-app-coordination/blob/master/CLA.md

### CODE REVIEW

File: packages/cms/src/desk-structure.ts [hunk 1]
```diff
@@ -25,6 +25,10 @@ const hiddenDocTypes = [
   'deceasedPage',
   'situationsPage',
   'hospitalPage',
+  'nursinghomePage',
```
Comment (VWSCoronaDashboard8) [inline packages/cms/src/desk-structure.ts:28]: I don't think nursing home is one word

File: packages/app/src/pages/landelijk/thuiswonende-ouderen.tsx [hunk 2]
```diff
@@ -36,13 +40,19 @@ export const getStaticProps = createGetStaticProps(
   selectNlPageMetricData(),
   createGetChoroplethData({
     vr: ({ elderly_at_home }) => ({ elderly_at_home }),
+  }),
+  createGetContent<{
+    articles?: ArticleSummary[];
+  }>((_context) => {
+    const locale = process.env.NEXT_PUBLIC_LOCALE || 'nl';
+    return createPageArticlesQuery('elderlyAtHomePage', locale);
   })
```
Comment (VWSCoronaDashboard8) [inline packages/app/src/pages/landelijk/thuiswonende-ouderen.tsx:49]:
```suggestion
  }>(() => {
    const locale = process.env.NEXT_PUBLIC_LOCALE || 'nl';
    return createPageArticlesQuery('elderlyAtHomePage', locale);
  })
```

Comment (VWSCoronaDashboard8) [reply packages/app/src/pages/landelijk/thuiswonende-ouderen.tsx:49]: But I see that _context is used in these functions in lots of places. Is there an idea behind it that I'm missing?

**Review by VWSCoronaDashboard8 — CHANGES_REQUESTED** [review]

File: packages/app/src/pages/landelijk/thuiswonende-ouderen.tsx (hunk 2 above)
Comment (VWSCoronaDashboard1) [reply packages/app/src/pages/landelijk/thuiswonende-ouderen.tsx:49]: underscoring it will stop the linter from yelling about an unused param

Comment (VWSCoronaDashboard1) [reply packages/app/src/pages/landelijk/thuiswonende-ouderen.tsx:49]: oh wait a sec, TS allows it to be omitted when unused. I'll make a change.

**Review by VWSCoronaDashboard8 — APPROVED** [review]

**Review by VWSCoronaDashboard14 — APPROVED** [review]

**Pull request merged**